apis/nucleus/src/components/listbox/hooks/useExistingModel.jsx

```diff
@@ -15,28 +15,30 @@ export default function useExistingModel({ app, qId, options = {} }) {
   }
 
   useEffect(() => {
-    function storeModel(m) {
-      if (modelStore.get(m.id)) {
-        return;
-      }
-      // Nebula gets the model from this store in some of its
-      // core functions, so it must always be stored here.
-      modelStore.set(m.id, m);
-      m.once('closed', () => {
-        modelStore.clear(m.id);
-      });
-    }
-
+    const handler = { cleanup: () => {} };
```

Comment:
(not really important, but)
why an object an not just a function value
ex:
```suggestion
    let cleanupFn = () => {};
```
or
```suggestion
    let cleanupFn = undefined;
```